celery/backends/dynamodb.py

```diff
@@ -223,6 +229,43 @@ def _wait_for_table_status(self, expected='ACTIVE'):
             achieved_state = current_status == expected
             sleep(1)
 
+    def _update_table_ttl(self):
```

Comment:
I think this operation should be optional, since some users may want to retain results.